packages/grafana-ui/rollup.config.ts

```diff
@@ -44,6 +45,9 @@ const buildCjsPackage = ({ env }) => {
       resolve(),
       image(),
       env === 'production' && terser(),
+      webWorkerLoader({
+        pattern: /(.+\.worker\.js$)/,
+      }),
     ],
```

Comment:
Not sure how this loader works. I don't like the duplication though of similar loaders in grafana-ui and in the core.

In https://github.com/grafana/grafana/pull/33803 and in https://github.com/grafana/grafana/pull/33800 we are moving some of the visualizations out from grafana-ui to core to keep things simplere, especially that those abstractions are not intended to be used outside of Grafana. Following the same pattern, I think we can move NodeGraph and it's relatives out from grafana-ui to NodeGraphPanel and thank't to that we could remove this config from rollup.

Or - is the NodeGraph component intended to be used by plugin devs?

Reply:
Makes sense in that case, it was here just because I followed some other visualisations that were there. Will do that after this is reviews though so it does not complicate the diff.